rholang/src/main/scala/coop/rchain/rholang/build/CompiledRholangSource.scala

```diff
@@ -59,8 +59,8 @@ object CompiledRholangTemplate {
       case (content, (name, value)) => content.replace(s"""$$$$$name$$$$""", value.toString)
     }
 
-    s"""//Loaded from resource file <<$classpath>>
-       #$finalContent
+    s"""$finalContent
+       # //Loaded from resource file <<$classpath>>
        #""".stripMargin('#')
```

Comment:
put the added comment in the end because it would trouble on debugging the content lines.

Reply:
What kind of troubles you had? Can you add additional comment at the end like:
`// End loaded from resource file <<$classpath>>`

Reply:
```
//Loaded from resource file <<$classpath>>
@a!(1)
```
The above codes would give error `Top level free variables are not allowed: a at 2:2.` while the actual error line is `1:2`